<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 237, 201);
selection-color: rgb(255, 255, 255);
selection-background-color: rgb(105, 109, 88);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип мероприятия:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Century Gothic&quot;;
background-color: rgb(105, 109, 88);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(254, 253, 239);
font: 9pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
